src/DependencyInjection/ContaoCoreExtension.php

```diff
@@ -45,6 +46,17 @@ public function getAlias()
     /**
      * {@inheritdoc}
      */
+    public function getConfiguration(array $config, ContainerBuilder $container)
+    {
+        // Will add the resource to the container
+        parent::getConfiguration($config, $container);
```

Comment:
That's still necessary. We don't need to add the YAML resources, but still need to add the `Configuration` class.